src/Draco.Lsp/Server/ILanguageServerLifecycle.cs

```diff
@@ -0,0 +1,17 @@
+using Draco.Lsp.Attributes;
+using Draco.Lsp.Model;
+using System.Threading.Tasks;
+
+namespace Draco.Lsp.Server;
+
+public interface ILanguageServerLifecycle
```

Comment:
Why did this become a public interface? The only thing that would theoretically need to implement this is `LanguageServerLifecycle`, which is the internal type dealing with registration and capabilities